src/App.js

```diff
@@ -1,16 +1,68 @@
 import React from 'react';
+import TodoForm from './TodoForm.js';
+
+
+
+const todos = [ {
+task: 'Organize Garage',
+id: Date.now,
+completed: false
+},
```

Comment:
So when you are populating a list of dummy data for a starting place, create a hard coded id instead of having a bunch of items starting with the Date.now() method. With the speed of the browsers now it is possible in the map to generate 2 or more items within a millisecond and create duplicate ids